services/web/project/static/swagger.yml

```
@@ -0,0 +1,275 @@
+        address:
+          type: string
+          description: The contact's address
+          
+        description:
+          type: string
+          description: The contact's description
+        
+        tags:
+          type: string
+          description: An array of tags for the contact
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)